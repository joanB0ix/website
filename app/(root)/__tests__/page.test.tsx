import { render } from "@testing-library/react";

import Home from "../page";

jest.mock("../about/WhoamiSection");
jest.mock("../about/SocialsSection");
jest.mock("../experience/ExperienceSection");
jest.mock("../projects/ProjectsSection");

describe("Home component snapshot", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
