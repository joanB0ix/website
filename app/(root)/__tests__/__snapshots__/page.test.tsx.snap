// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component snapshot renders correctly and matches snapshot 1`] = `
<div>
  <main
    class="flex-grow"
  />
</div>
`;
